modules/rostests/apitests/ntdll/NtLoadUnloadKey.c

```diff
@@ -393,6 +393,7 @@ DisconnectRegistry(
                                RootKey,
                                NULL);
     // return NtUnloadKey(&ObjectAttributes);
+// ToDo: This single NtUnloadKey2 is >= 5.2.
     return NtUnloadKey2(&ObjectAttributes, Flags);
 }
```

Comment:
@HBelusca, would there be a minimal change, like conditionally using NtUnloadKey() or skipping part of the test?
Or should the whole test be skipped?

Reply:
Better would be to do it conditionally (using a delay-load or whatever), depending on the OS version, and if falling back to the NtUnloadKey case, TRACEing some message in addition.

Reply:
I now tested this locally, as current code is:

`DPRINT1()`: outputs nothing.
`TRACE()`: "error: implicit declaration of function 'TRACE'".
`printf()`: works.
`win_skip()`: works, as an example.

This ApiTest is the only one to define/use `DPRINT1()`.
It seems that part should be documented. Or even rewritten with something that works and is less confusing?

Reply:
XP is not a priority. Just do ANYTHING to make it work. You don't need comments, you don't need complicated conditionals. Either skip this function call or skip the whole test, but stop discussing it.
This whole PR should be 5 minutes of work, you're dragging it out into hours, and there are 10000 more important things to work on.